pages/tools/fileconverter.vue

```diff
@@ -286,6 +307,12 @@ export default {
         this.acceptedExtensions = extensions.join(',')
       })
     },
+    GetPackagesVersions() {
+      this.$axios.get(`${this.path}/versions`).then((response) => {
```

Comment:
If you do not handle error, you can simplify the axios calls with the $get or $post
https://axios.nuxtjs.org/usage#component